soc/cypress/Kconfig.soc

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2018, Cypress
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_PSOC6
+	bool
+
+config SOC_FAMILY_INFINEON_CAT1
+	bool
+
+config SOC_FAMILY_INFINEON_CAT1A
+	bool
+
+config SOC_FAMILY
+	default "psoc6" if SOC_FAMILY_PSOC6
+	default "infineon_cat1" if SOC_FAMILY_INFINEON_CAT1
```

Comment:
what about CAT1A?

Reply:
This soc was created in a mess, cat and cat1a are always selected, it does something in their HAL